app/src/main/res/values/dimens.xml

```diff
@@ -7,4 +7,5 @@
     <dimen name="sp_15">15sp</dimen>
     <dimen name="sp_13">13sp</dimen>
     <dimen name="sp_18">18sp</dimen>
+    <dimen name="dp_100">100dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_100` appears to be unused
 Unused resources make applications larger and slow down builds.

Reply:
Em dùng cho pull sau ạ.